configuration/NI.Analysis.targets

```diff
@@ -0,0 +1,110 @@
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+
+  <PropertyGroup>
+    <!-- TODO: This location of 'content' needs to get updated once we build our package if we decide to put the files elsewhere, outside of a 'content' directory -->
+    <CodeAnalysisRuleSetDirectory>$(PkgNI_DotNet_Analyzers)\content</CodeAnalysisRuleSetDirectory>
+    <NuGetPackagesDirectory>$(NuGetPackageRoot)</NuGetPackagesDirectory>
+  </PropertyGroup>
+
+	<PropertyGroup>
```

Comment:
Mix of tabs and spaces in this file :)

Replace with spaces, 2 or 4, preferably 4.

Reply:
Replaced with 2 spaces which matches our ASW conventions for XML config files.